README

```diff
@@ -523,15 +523,21 @@ MPI Functionality and Features
       deprecated over 20 years ago, and finally removed from the MPI
       specification in MPI-3.0 (in 2012).
 
-  *** The Open MPI FAQ (https://www.open-mpi.org/faq/) contains
-      examples of how to update legacy MPI applications using these
-      deleted symbols to use the "new" symbols.
+  *** The Open MPI FAQ (https://www.open-mpi.org/faq/?category=mpi-removed)
+      contains examples of how to update legacy MPI applications using
+      these deleted symbols to use the "new" symbols.
 
   All that being said, if you are unable to immediately update your
   application to stop using these legacy MPI-1 symbols, you can
   re-enable them in mpi.h by configuring Open MPI with the
   --enable-mpi1-compatibility flag.
 
+  NOTE: Open MPI v4.0.0 had an error where these declarations were not
+        included in the library if configured without --enable-mpi1-compatibility
```

Comment:
Is this right?

I don't remember the exact behavior of v4.0.0.  There are two things:

1. The declarations in `mpi.h` (and friends)
1. The symbols in `libmpi`

Which did `--enable-mpi1-compatibility` enable/disable in 4.0.0?  Your text says it was only `#1`.  Is that correct?

Reply:
In searching around https://github.com/open-mpi/ompi/issues/6114 would be a better Issue to reference from the README, describing what the issue with v4.0.0 was.
without `--enable-mpi1-compatibility` the header produced a deprecation (very confusing) warning message, and the symbols were un-intentionally removed from the library (i.e. link error).